### PR TITLE
Add new parameters to shortcode, and create a new shortcode

### LINKED ISSUE #576: Add a new `class` parameter for the `[wcpdf_download_invoice]` and `[wcpdf_download_pdf]` shortcodes
Currently, the `[wcpdf_download_invoice]` and `[wcpdf_download_pdf]` shortcodes return a link wrapped into a paragraph, with this pattern:

```html
<p><a href="%s" target="_blank">%s</a></p>
```

A customer ([here](https://wordpress.org/support/topic/shordcode-css-classes/)), proposed adding the ability to pass a class to the shortcode as a parameter. We should add the class to the paragraph to allow applying CSS rules to both the paragraph and/or the link.

### PR DESCRIPTION
**Pull request by MohamadNateqi**
closes #576

### CODE REVIEW

File: includes/class-wcpdf-frontend.php [hunk 1]
```diff
@@ -109,7 +110,7 @@ public function woocommerce_api_invoice_number ( $data, $order ) {
 		return $data;
 	}
 
-	public function generate_document_link_shortcode( $atts ) {
+	public function generate_document_link_shortcode( $atts, $content = null, $tag = '' ) {
```
Comment (alexmigf) [inline includes/class-wcpdf-frontend.php:113]: Maybe this function should be renamed to `generate_document_shortcode`?

Comment (alexmigf) [inline includes/class-wcpdf-frontend.php:113]: Maybe rename `$tag` as `$shortcode_tag`? Like they indicate in docs.

**Review by alexmigf — CHANGES_REQUESTED** [review]

**Review by alexmigf — APPROVED** [review]
LGTM!

**Review by YordanSoares — APPROVED** [review]
✅ LGTM!

**Pull request merged**